crates/context_aware_config/src/helpers.rs

```diff
@@ -38,6 +42,13 @@ use crate::db::{
     },
 };
 
+#[derive(Debug)]
+pub struct DimensionData {
+    pub schema: JSONSchema,
+    pub priority: i32,
+    pub position: i32,
```

Comment:
are we sure we want to call this `position` over here and not `weight` or something similar ?

Reply:
yes it will be position for dimension